src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -76,10 +84,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 feedbackResponseComment.showGiverNameTo.add(FeedbackParticipantType.valueOf(viewer.trim()));
             }
         }
-
+        FeedbackResponseCommentAttributes updatedComment = new FeedbackResponseCommentAttributes();
```

Comment:
This can be set to `null` since the `new FeedbackResponseCommentAttributes()` isn't used at all.